packages/devtools_app/lib/src/shared/analytics/constants/_network_constants.dart

```diff
@@ -0,0 +1,5 @@
+part of '../constants.dart';
```

Comment:
This file needs the following copyright header:


```
// Copyright 2024 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
```

Reply:
added the copyright header